google-cloud-tools-plugin/src/com/google/cloud/tools/intellij/GctFeature.java

```diff
@@ -24,16 +24,34 @@
 
 /**
  * The set of Google Cloud Tools features that we want to be able to conditionally enable.
+ *
+ * <p>Enum elements marked {@link Deprecated} are fully released and no longer hidden behind a flag.
  */
 public enum GctFeature implements Feature {
+  @Deprecated
   APPENGINE_FLEX(null, "feature.appengine.flex", "ct4ij.feature.appengine.flex");
```

Comment:
@joaoandremartins nothing. The main reason I left it is that it serves as an example, provides history, and the enum looks weird with no instances.  I'm on the fence about keeping the old ones around.